src/App.js

```diff
@@ -1,9 +1,15 @@
+import React, { Component } from 'react';
 import './App.css';
+import Person from './Person/Person.js';
 
 function App() {
   return (
     <div className="App">
       <h1>React App</h1>
+      <p>This is really working</p>
+      <Person name="Max" age="28" />
+      <Person name="Manu" age="29" >My hobbies: Racing</Person>
+      <Person name="Stepha" age="26" />
```

Comment:
Please update your layout to correspond to requested in task assignment header and card positioning with applying minimum styling via css as requested - separate **Header** (better create separate component) and separate **Card**(separate component, with caption and text body passed as props or children nodes):
![image](https://user-images.githubusercontent.com/52723192/103827630-7abf5300-508a-11eb-8d9a-cc3b69ba5a72.png)